### ISSUE TITLE
Problem with validation loss 

### ISSUE THREAD

**Issue by isaacmg** [AIStream-Peelout/flow-forecast#203]
So looks like I f**ked something up when added the compute loss function https://github.com/AIStream-Peelout/flow-forecast/blob/e0155aac47837c885690bb952e2e45e15afaee2f/flood_forecast/pytorch_training.py#L153

Validation loss is not changing

**Comment by isaacmg** [comment]
<img width="441" alt="image" src="https://user-images.githubusercontent.com/3865062/99922392-c00c0900-2d06-11eb-8cf9-06215977cc90.png">

**Comment by isaacmg** [comment]
Problem with MASE actually.
